raiden/network/transport/matrix/transport.py

```diff
@@ -195,20 +201,23 @@ def enqueue(self, queue_identifier: QueueIdentifier, messages: List[Message]) ->
                         message=message,
                         text=MessageSerializer.serialize(message),
                         expiration_generator=expiration_generator,
+                        address_metadata=address_metadata,
                     )
                     encoded_messages.append(data)
 
             self._message_queue.extend(encoded_messages)
 
         self.notify()
 
-    def enqueue_unordered(self, message: Message) -> None:
+    def enqueue_unordered(
+        self, message: Message, address_metadata: AddressMetadata = None
```

Comment:
I think it must be `Optional[AddressMetadata]` here

Reply:
Our dev guidelines suggest to omit the `Optional` type-hint when the default value is already defined as None